contributors/devel/sig-instrumentation/migration-to-structured-logging.md

```diff
@@ -378,32 +379,27 @@ func ChangePodStatus(newStatus, currentStatus string) {
 [lowerCamelCase]: https://en.wiktionary.org/wiki/lowerCamelCase
 [alphanumeric]: https://en.wikipedia.org/wiki/Alphanumeric
 
-### Use `klog.KObj` and `klog.KRef` for Kubernetes objects
+## Ensure value is properly passed
```

Comment:
```suggestion
## Good practice for passing values in structured logging
```
?

Reply:
Not sure about calling this a good practice as this should be a required step for migration to change how values are provided.